src/Engine/MapController.java

```diff
@@ -110,10 +114,10 @@ public boolean handleInput(InputHandler.InputAction input)
         exitMap = handleConfirmExitMenuInput(input);
         break;
       case CO_INFO:
-        if( coInfoMenu.handleInput( input ) )
+        if( coInfoMenu.handleInput(input) )
```

Comment:
There seem to be a lot of whitespace changes. Do we have conflicting autoformat settings?

Reply:
The file `DefPcCodeStyle.xml` at the top level of the repo contains the code settings I thought we both were using. You should set Eclipse to use this file for this project so we don't keep stepping on each other.

Reply:
I already am using that file. That's why you never see any braces like
if (thing) {
   then do this
}